### DISCUSSION TITLE
Jellyseer Email SMTP

### DISCUSSION THREAD

**Discussion by pcmodder2001** [seerr-team/seerr#334] [Q&A] [answered]
Hiya, 

Im looking for some help setting up Email notifactions on jellyseer. 

im using gmail SMTP, im just a little confused as what the sender address, SMTP host, SMTP username / password should be? 

Please could someone give me some advice on getthing this set up ?

**Comment by fallenbagel** [comment, marked as answer]
Your sender address will be your email,
For example
`sender address: pcmodder2001@gmail.com`
`SMTP Host: smtp.gmail.com`

Your smtp username will be the email you're using for smtp which will probably also be your email
`smtp username: pcmodder2001@gmail.com`

As for your password, you'll have to go to your gmail settings on web, go to security and you should see something called app password, select custom and write `jellyseerr` as name and copy the password it generates. You'll be using this password in the password field.

**Reply by Uganto** [reply]
Do you know how to build an SMTP?

**Reply by fallenbagel** [reply]
> Do you know how to build an SMTP?

Do you mean like build a mail server? 🤔

**Comment by pcmodder2001** [comment]
Awesome thank you 😊

Next question would be, on the other tabs I can see you can select what
notifications you want to be sent. But for email there isn't that option.
Or is everything just emailed?

**Reply by fallenbagel** [reply]
The selection of that for users is in their own profile settings > notifications

**Comment by pcmodder2001** [comment]
Also, just learned this isn't sending my users emails only the admins

**Reply by fallenbagel** [reply]
Did you setup smtp in your Gmail?

**Comment by pcmodder2001** [comment]
Hiya,

Sorry had a play abit more and all sorted now thank you :)
